docs/_static/link_gen/link.js

```diff
@@ -58,22 +84,52 @@ var apps = {
     }
 }
 
+function changeTab(div) {
+    var hub = document.getElementById("hub");
+    var env_repo = document.getElementById("repo");
+    var content_repo = document.getElementById("content-repo");
+    var content_branch = document.getElementById("content-branch");
+    var id = div.id;
+
+    if (id.includes("mybinder")) {
+        hub.placeholder = "https://mybinder.org";
```

Comment:
Maybe set it to default value, so folks don't have to manually edit it?

Do you think we can also change the label to say 'BinderHub URL'?

Reply:
Also, the label under 'hub name' says 'hub must have nbgitpuller installed', but in the mybinder.org situation, the *environment repo* must have nbgitpuller installed.